extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/BackChannelLogoutHandler.java

```diff
@@ -46,7 +46,7 @@ public void setup(@Observes Router router) {
 
     private void addRoute(Router router, OidcTenantConfig oidcTenantConfig) {
         if (oidcTenantConfig.isTenantEnabled() && oidcTenantConfig.logout.backchannel.path.isPresent()) {
-            router.route(getRootPath() + oidcTenantConfig.logout.backchannel.path.get())
+            router.route(oidcTenantConfig.logout.backchannel.path.get())
```

Comment:
Who is calling this method? Which router is it? Why it's not a route registered in the processor?

Reply:
Hi @cescoffier 

>Who is calling this method?

It is called from just above from `setup(@Observes Router router) `, [here](https://github.com/quarkusio/quarkus/blob/228883027f96ba285a1ff9e75e7bacc3369dc041/extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/BackChannelLogoutHandler.java#L39).

>Which router is it?

The main application one, since it works when accessed from the tests and for example, from Keycloak.

>Why it's not a route registered in the processor?

Do you mean the recorder, similar to how it is done [WebAuthnRecorder](https://github.com/quarkusio/quarkus/blob/main/extensions/security-webauthn/runtime/src/main/java/io/quarkus/security/webauthn/WebAuthnRecorder.java#L35) ?

I don't recall now, probably I did not know how to do it at a time, and chose `setup(@Observers Router router)`, or may be because the way it is done now is easier to handle in a multi-tenant setup. I can certainly rework it and move the registration to the recorder. I guess, to have this PR backportable, I should do it as a follow up PR a bit later, does it sound reasonable ?

Reply:
Yeah, please let's try to get the simple version of this PR verified, tested and merged.

I need a backportable fix very soon.